examples/tech-demos/21-example-technology-showcase-buckling.py

```diff
@@ -0,0 +1,415 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Sep  2 14:24:13 2022
+
+@author: pmaroneh
+"""
+
+""".. _ref_buckling_postbuckling_ring_stiffened_cylinder:
+
+Buckling and post-buckling analysis of a ring-stiffened
+cylinder using nonlinear stabilization
+======================================================
+
+This examples shows how to use PyMAPDL to import an existing FE model and to
+perform a nonlinear buckling and postbuckling analysis using nonlinear
+stabilization. The problem uses a stiffened cylinder subjected to uniform
+external pressure to show how to find the nonlinear buckling loads,
+achieve convergence at the post-buckling stage, and interpret the results.
```

Comment:
As explained before, this file will need to be changed into a RST file. Let me know if you have any question about that.

Reply:
@clatapie I have no clue how to do that and feel like giving up on this one. 
@germa89 fyi

Reply:
We'll find a solution, you created a great example! 😄  
I contacted you on Teams to arrange a call about that if you want.